tests/test_utilities.py

```diff
@@ -503,7 +503,7 @@ def test_send_email_notification_message(
                 assert getattr(
                     mail.outbox[0], field
                 ) == val, f'Did not match attrib: {field} for {enterprise_customer_name}'
-            assert mail.outbox[0].connection is conn
+            assert isinstance(mail.outbox[0].connection, type(conn))
```

Comment:
Django 5.2 has renamed the underlying tuples to named tuples which means the underlying resource to `msg.send` is no longer the same object but only of the same type.

> Tuple items of [EmailMessage.attachments](https://docs.djangoproject.com/en/5.2/topics/email/#django.core.mail.EmailMessage) and [EmailMultiAlternatives.attachments](https://docs.djangoproject.com/en/5.2/topics/email/#django.core.mail.EmailMultiAlternatives) are now named tuples, as opposed to regular tuples.